roles/kubernetes/preinstall/tasks/0020-verify-settings.yml

```diff
@@ -161,6 +161,12 @@
     - cloud-provider
     - facts
 
+- name: Make sure we have an IPv6 address on all nodes
+  assert:
+    that: "{{ ansible_default_ipv6.address is defined and ansible_default_ipv6.scope == 'global' }}"
```

Comment:
`that: ansible_default_ipv6.address is defined and ansible_default_ipv6.scope == 'global'` ?

Reply:
Original commit updated